app/gitcoinbot/actions.py

```diff
@@ -20,69 +20,161 @@
+                           'TIME & [more](https://github.com/gitcoinco/web/blob/master/app/dashboard/tokens.py)'])
     help_text_response = f"I am {github_bot_user}, a bot that facilitates gitcoin bounties.\n" \
         "\n<hr>Here are the commands I understand:\n\n " \
-        "* `bounty <amount> ETH` -- receive link to gitcoin.co form to create bounty.\n " \
-        "* `claim` -- receive link to gitcoin.co to start work on a bounty.\n " \
-        "* `tip <user> <amount> ETH` -- receive link to complete tippping another github user *<amount>* ETH.\n " \
+        "* `bounty <amount> <currency>` -- receive link to gitcoin.co form to create bounty.\n " \
+        "* `submit work` -- receive link to gitcoin.co to start work on a bounty.\n " \
+        "* `tip <user> <amount> <currency>` -- receive link to complete tippping another " \
+                         "github user *<amount>* <currency>.\n " \
         "* `help` -- displays a help menu\n\n<br>" \
+        f"Some currencies I support: \n{currencies}\n\n<br>" \
         "Learn more at: [https://gitcoin.co](https://gitcoin.co)\n" \
         f":zap::heart:, {github_bot_user}\n"
     return help_text_response
 
 
+@Bound(amount_greater_than_zero)
 def new_bounty_text(owner, repo, issue_id, comment_text):
+    """Based on comment_text generate instructions to create a bounty with the specified amount and token"""
     issue_link = f"https://github.com/{owner}/{repo}/issues/{issue_id}"
     bounty_amount = parse_comment_amount(comment_text)
-    bounty_link = f"{settings.BASE_URL}funding/new?source={issue_link}&amount={bounty_amount}"
+    token_name = parse_comment_currency(comment_text)
+    bounty_link = f"{settings.BASE_URL}funding/new?source={issue_link}&amount={bounty_amount}&tokenName={token_name}"
     new_bounty_response = f"To create the bounty please [visit this link]({bounty_link}).\n\n " \
-        "PS Make sure you're logged into Metamask!"
+                          "PS Make sure you're logged into Metamask!"
     return new_bounty_response
 
 
+def no_active_bounty(owner, repo, issue_id):
+    """Instruct to create a bounty"""
+    issue_link = f"https://github.com/{owner}/{repo}/issues/{issue_id}"
+    bounty_link = f"{settings.BASE_URL}funding/new?source={issue_link}"
+    no_active_bounty_response = "No active bounty for this issue, consider create the bounty please " \
+                                f"[visit this link]({bounty_link}).\n\n " \
+                                "PS Make sure you're logged into Metamask!"
+
+    return no_active_bounty_response
+
+
 def parse_comment_amount(comment_text):
     re_amount = r'\d*\.?\d+'
     result = re.findall(re_amount, comment_text)
     return result[0]
 
 
+def parse_comment_currency(comment_text):
+    """Get the first token defined in comment_text"""
+    or_currencies = r'|'.join(CURRENCIES)
+    re_currencies = r'({})'.format(or_currencies)
+    result = re.findall(re_currencies, comment_text)
+
+    return result[0]
```

Comment:
What if no currency/match is found?  Looks like it'd result in an `IndexError`.

Reply:
Right :thinking:, the result now will be fallback this to a defined default currency.